Example/Pods/TextAnnotation/TextAnnotation/Classes/CustomSwitch.swift

```diff
@@ -0,0 +1,278 @@
+import Foundation
+
+public class CustomSwitch: NSControl {
+  public static let bundle = Bundle(for: CustomSwitch.self)
+  
+  public var tintColor = NSColor(calibratedRed: 0.243, green: 0.235, blue: 0.243, alpha: 1.0) {
```

Comment:
@dworak is there any difference between `calibratedRed` and `red`? Any reasons why do you use `calibratedRed` here and other calibrated ones in other places?